package.json

```diff
@@ -6,7 +6,7 @@
     "displayName": "Azure Databases",
     "description": "Create, browse, and update globally distributed, multi-model databases in Azure.",
     "engines": {
-        "vscode": "^1.40.0"
+        "vscode": "^1.50.0"
```

Comment:
Not quite - when you update the VS Code engine, you always want to be on the _minimum_ version you need (in our case 1.48.0). Otherwise, people on older versions of VS Code won't get the update to our extension until they update VS Code, which just slows down the adoption of our latest extension